### ISSUE TITLE
Pull Request URLs are incorrect for repositories with "limited refs" enabled

### ISSUE THREAD

**Issue by hross** [Microsoft/vsts-vscode#244]
I noticed that all of the PR URLs generated from the extension are wrong (separate PR in flight to fix that).

As part of that, the reason is we check urls like this:

https://mseng.visualstudio.com/VSOnline/_git/VSO/vsts/info

Which gives back a remoteUrl that contains a non-limited ref version of the repository URL... i.e. /_git/_full/.

This is okay in certain instances but we also use it to generate URLs for version control, which are wrong because they have _full in them, which is interpreted as project name.

I'm not sure of the implications but you could just replace /_git/_full/ with /_git/ in RepositoryInfo.ts:

```
     if (typeof repositoryInfo === "object") {
            repositoryUrl = repositoryInfo.repository.remoteUrl;
```

I didn't do that because I wasn't sure of the implications, so I filed this issue instead.

**Comment by jeffyoung** [comment]
I'm a bit confused @hross.  Did something on the server break compatibility/change?  Is that what you're referring to?  I'll have to look into it but if you've got more details, that'd be helpful.

**Comment by jeffyoung** [comment]
The upshot here is that the pull request URLs are incorrect for repositories that have the "limited refs" capability enabled.  Here's a document I found talking about [limited refs](https://www.visualstudio.com/learn/limited-refs/).

**Comment by jeffyoung** [comment]
Addressed this in v1.119.0.
